google/cloud/opentelemetry/internal/recordable.cc

```diff
@@ -148,8 +150,28 @@ google::devtools::cloudtrace::v2::Span&& Recordable::as_proto() && {
 }
 
 void Recordable::SetIdentity(
-    opentelemetry::trace::SpanContext const& /*span_context*/,
-    opentelemetry::trace::SpanId /*parent_span_id*/) noexcept {}
+    opentelemetry::trace::SpanContext const& span_context,
+    opentelemetry::trace::SpanId parent_span_id) noexcept {
+  std::array<char, 2 * opentelemetry::trace::TraceId::kSize> hex_trace_buf;
+  span_context.trace_id().ToLowerBase16(hex_trace_buf);
+  std::string const hex_trace(hex_trace_buf.data(),
+                              2 * opentelemetry::trace::TraceId::kSize);
```

Comment:
`hex_trace_buf.size()` perhaps, to avoid repeating the template parameter, and to perpetuate the "data/size" pairing convention.

Reply:
Good idea. Fixed.